content/post/kubestash-v2024.12.9/index.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Introducing KubeStash v2024.12.9
+date: "2024-12-09"
+weight: 10
+authors:
+- Md Ishtiaq Islam
+tags:
+- backup
+- backup-verification
+- disaster-recovery
+- kubernetes
+- kubestash
+- restore
+---
+
+We are pleased to announce the release of [KubeStash v2024.12.9](https://kubestash.com/docs/v2024.12.9/setup/), packed with new features and important bug fixes. You can check out the full changelog [HERE](https://github.com/kubestash/CHANGELOG/blob/master/releases/v2024.12.9/README.md).
+
+### New Features
+
+Here, we are going to highlight the new features that have been introduced in this release.
+
+#### KubeDB Managed Database Backup Verification
+
+We've introduced functionality to verify KubeDB managed database backup. You can configure following types of backup verification strategies:
+
+- **RestoreOnly :** KubeStash operator will initiate a `RestoreSession` using the addon information specified in `BackupVerifier`. The verification of the backup will rely on the status of the `RestoreSession` phase; if the restore completes successfully, the backup is considered verified.
+- **Query :** At first, KubeStash operator will initiate a `RestoreSession` and after successful restore, it will create a verifier job to run the queries provided in `BackupVerifier`.
+- **Script :** At first, KubeStash operator will initiate a `RestoreSession` and after successful restore, it will create a verifier job to run the script provided in `BackupVerifier`.
+
+Here we will give an example of query verification strategy for KubeDB managed `MySQL`.
+
+At first, we need to check if the verification function is present in our cluster using the following command:
+
+```bash
+$ kubectl get functions | grep -i `kubedbverifier`
+kubedbverifier            19h
+```
+
+Assuming that a KubeDB managed `MySQL` instance with a configured backup already exists. Now, we need to create a `MySQL` instance for verification in our preferred namespace (i.e. `verify`). Here is an example of `MySQL` instance for verification:
```

Comment:
To verify the backup of a MySQL database, a separate MySQL instance  is required to run the verification process for the original database.  Here is an example of a MySQL instance for verification: